.babelrc

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
- [ ] In your readme you have all the required sections but the instructions are not clear. You have missed instructions. Kindly resolve the issue.